<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PET_device_info_page</class>
 <widget class="QWidget" name="PET_device_info_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Состояние устройства</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Производитель</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Версия аппаратного обеспечения</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Версия программного обеспечения</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Уникальный идентификатор (UID)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="le_hw_version"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Тип устройства</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="le_vid"/>
     </item>
     <item row="12" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
     </item>
     <item row="8" column="2">
      <widget class="QComboBox" name="cb_state_dev"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="le_pid"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="le_prot_version"/>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Опорная частота, МГц</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Время работы устройства</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Версия  протокола ПЭТ</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="le_uid"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="le_sw_version"/>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="le_work_time"/>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Режим синхронизации</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Период синхронизации, с</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Контроль периода синхронизации</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QComboBox" name="cb_sync_mode"/>
     </item>
     <item row="11" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Информация об устройстве</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметры синхронизации</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
